nav.go

```diff
@@ -525,53 +548,46 @@ func (nav *nav) preview(path string, win *win) {
 	}
 }
 
-func (nav *nav) previewClear() {
-	if len(gOpts.cleaner) != 0 && nav.volatilePreview {
-		nav.volatilePreview = false
+func (nav *nav) preview(path string) {
```

Comment:
Maybe get rid of single line function here?

Reply:
OK, done.